---
# Leave the homepage title empty to use the site title
title: ''
date: 2022-10-24
type: landing

sections:
  - block: hero
    demo: true # Only display this section in the Wowchemy demo site
    content:
      title: Hugo Academic Theme
      image:
        filename: hero-academic.png
      cta:
        label: '**Get Started**'
        url: https://wowchemy.com/templates/
      cta_alt:
        label: Ask a question
        url: https://discord.gg/z8wNYzb
      cta_note:
        label: >-
          <div style="text-shadow: none;"><a class="github-button" href="https://github.com/wowchemy/wowchemy-hugo-themes" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star">Star Wowchemy Website Builder</a></div><div style="text-shadow: none;"><a class="github-button" href="https://github.com/wowchemy/starter-hugo-academic" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star">Star the Academic template</a></div>
      text: |-
        **Generated by Wowchemy - the FREE, Hugo-based open source website builder trusted by 500,000+ sites.**

        **Easily build anything with blocks - no-code required!**

        From landing pages, second brains, and courses to academic resum√©s, conferences, and tech blogs.

        <!--Custom spacing-->
        <div class="mb-3"></div>
        <!--GitHub Button JS-->
        <script async defer src="https://buttons.github.io/buttons.js"></script>
    design:
      background:
        gradient_end: '#1976d2'
        gradient_start: '#004ba0'
        text_color_light: true
  - block: about.biography
    id: about
    content:
      title: 
      # Choose a user profile to display (a folder name within `content/authors/`)
      username: admin
  #- block: skills
  #  content:
  #   title: Skills
  #   text: ''
     # Choose a user to display skills from (a folder name within `content/authors/`)
  #    username: admin
  #  design:
  #   columns: '1'
  - block: experience
    content:
      title: Experience
      # Date format for experience
      #   Refer to https://wowchemy.com/docs/customization/#date-format
      date_format: Jan 2006
      # Experiences.
      #   Add/remove as many `experience` items below as you like.
      #   Required fields are `title`, `company`, and `date_start`.
      #   Leave `date_end` empty if it's your current employer.
      #   Begin multi-line descriptions with YAML's `|2-` multi-line prefix.
      items:
        - title: Post-baccalaureate researcher
          company: Northwestern University, Chicago
          company_url: 'https://www.feinberg.northwestern.edu/index.html'
          company_logo: org-gc
          location: Illinois
          date_start: '2022-11-15'
          date_end: ''
          description: |2-
              Projects:

              * **Automated flow cytometry pipeline:** <br> 
                
                 Led a year-long [project](https://docs.google.com/presentation/d/10BWF-NMwT0HdYpbd-NxwFP6GKXvv9o0qpw4vTQvvwDQ/edit?usp=sharing) to implement an innovative automated flow cytometry data analysis pipeline using gradient boosting models, significantly reducing data analysis time for wet bench researchers. <br> By automating the data processing and classification steps, the pipeline improved workflow efficiency and allowed for quicker access to critical insights. This work was used for multiple projects within the laboratory. First author paper based on this work is in progress.
              * **Differential gene expression methods benchmarking:** <br>
                Worked with [computational biology group](https://www.mimuw.edu.pl/~szczurek/) from the university of Warsaw on a large collaborative project focusing on the analysis of single cell RNA-seq data from hundreds of patients.<br> Benchmarked traditional approaches for differential gene expression methods vs deep learning-based approaches. The results of this work were essential for downstream tasks in this project. Publication is expected in 2024.
              * **Single cell spatial transcriptomics analysis:** <br>
                [Researched](https://docs.google.com/presentation/d/1l1zRh1nwhJxOKwIodAbxsXKHls5V1DgyfBAeG93__zI/edit?usp=sharing) spatial transcriptomics platforms (Nanostring CosMx, 10x Xenium) that helped bringing these technologies to the lab. Utilized various best-practice computational tools and techniques to decipher spatial patterns and identify potential biomarkers associated with pulmonary diseases, assessed publicly available datasets.
              * **Single cell RNA-seq data analysis:**<br>
               Analyzed single-cell data from pulmonary samples to gain insights into cellular heterogeneity and molecular changes in disease conditions. Employed bioinformatics tools to preprocess and interpret single-cell RNA sequencing data, revealing critical cellular subpopulations and their functional roles.
        - title: Junior Bioinformatician
          company: BostonGene Corporation, Boston, MA
          company_url: 'https://bostongene.com'
          company_logo: org-x
          location: Massachusetts 
          date_start: '2021-01-01'
          date_end: '2022-11-15'
          description: |2-
              Projects:
        
              * **Cell type deconvolution of cancer- associated fibroblasts in bulk RNA-seq data using machine learning methods with the goal to improve outcome prediction in cancer patients:**<br>
                Actively contributed to the development of a project on classifying fibroblasts in various tumors' microenvironment using machine learning algorithms, which resulted in my thesis. Work on this project was continued and results were submitted to [AACR](https://www.aacr.org/meeting/aacr-annual-meeting-2024/).
              * **Machine learning models validation:**<br>
                Developed a script to validate machine learning models that deconvolve sample cells' percentages based on sequencing data.
              * **Enhanced cross-team communication:**<br>
                Developed standard report form of RNA samples analysis to share results efficiently across multiple bioinformatics teams.
              * **Reproducible programming and RNA-seq analysis:**<br>
                Performed code debugging, analysis, and filtration of RNA-seq lab samples.
    design:
      columns: '2'
  - block: accomplishments
    content:
      # Note: `&shy;` is used to add a 'soft' hyphen in a long heading.
      title: 'Relevant courses and workshops'
      subtitle:
      # Date format: https://wowchemy.com/docs/customization/#date-format
      date_format: Jan 2006
      # Accomplishments.
      #   Add/remove as many `item` blocks below as you like.
      #   `title`, `organization`, and `date_start` are the required parameters.
      #   Leave other parameters empty if not required.
      #   Begin multi-line descriptions with YAML's `|2-` multi-line prefix.
      items:
        - certificate_url: ''
          date_end: '2022-08-10'
          date_start: '2020-09-10'
          description: |2-
            Went through extensive selection (100/5000 people) and took classes from academia and industry leaders in:
            + Machine learning *(A. Djakonov, Lomonosov Moscow State University)*,
            + Data engineering *(P. Klemenkov, NVIDIA)*,
            + Statistics *(V. Panov, Computer science department, Higher School of Economics)*,
            + Linear algebra *(I. Oseledets, Skoltech)*,
            + Linux, Bash, and Git *(A. Trunov)*.
          organization: AI masters
          organization_url: https://aimasters.ru
          title: Machine Learning and Deep Learning courses
          url: ''
        - certificate_url: ''
          date_end: ''
          date_start: '2022-02-01'
          description: Computer science classes focusing on latest research in various topcis ranging from graph neural networks to use of neural networks in bioinformatics  
          organization: Computer Science department Higher School of Economics
          organization_url: ''
          title: Winter Computer Science School
          url: ''
        - certificate_url: https://stanislavbratchikov.github.io/uploads/quantitative_genetics_certificate.pdf
          date_end: '2021-12-21'
          date_start: '2021-09-01'
          description: Gained entry into genetics courses, being among the 35 students selected. Topics included Genes in populations, Complex traits genetics, Genome Wide Associated Studies.
          organization: Moscow Insitute of Physics and Technology
          organization_url: ''
          title: 'Quantative genetics course by prof. Yurii Aulchenko'
          url: ''
        - certificate_url: https://stanislavbratchikov.github.io/uploads/summer_bioinformatics_school.pdf
          date_end: ''
          date_start: '2021-08-01'
          description: |2-
            Took multiple workshops on multiple topics on the use of Machine learning in Bioinformatics from academia leaders:
            + Machine Learning for Molecule Data *(Jose Miguel, University of Cambridge)*,
            + Machine Learning in Protein Structural Bioinformatics (Cryo-EM and protein docking) *(Daisuke Kihara, Purdue University)*,
            + Biomedical text mining and its application *(Shannkai Yan, National Institutes of Health)*,
            + DNABERT: BERT models for genome DNA language *(Ramana Davuluri, Stony Brook University)*.
          organization: Computer Science department Higher School of Economics
          organization_url: ''
          title: 'Summer School on Machine Learning in Bioinformatics'
          url: ''

    design:
      columns: '2'
  #- block: collection
  #  id: posts
  #  content:
  #    title: Recent Posts
  #    subtitle: ''
  #    text: ''
  #    # Choose how many pages you would like to display (0 = all pages)
  #    count: 5
  #    # Filter on criteria
  #    filters:
  #      folders:
  #        - post
  #      author: ""
  #      category: ""
  #      tag: ""
  #      exclude_featured: false
  #      exclude_future: false
  #      exclude_past: false
  #      publication_type: ""
  #    # Choose how many pages you would like to offset by
  #    offset: 0
  #    # Page order: descending (desc) or ascending (asc) date.
  #    order: desc
  #  design:
  #    # Choose a layout view
  #    view: compact
  #    columns: '2'
  - block: portfolio
    id: projects
    content:
      title: Projects
      filters:
        folders:
          - project
      # Default filter index (e.g. 0 corresponds to the first `filter_button` instance below).
      default_button_index: 0
      # Filter toolbar (optional).
      # Add or remove as many filters (`filter_button` instances) as you like.
      # To show all items, set `tag` to "*".
      # To filter by a specific tag, set `tag` to an existing tag name.
      # To remove the toolbar, delete the entire `filter_button` block.
      buttons:
        - name: All
          tag: '*'
        - name: Flow cytometry
          tag: Flow cytometry
        - name: SC RNAseq
          tag: SC RNAseq
    design:
      # Choose how many columns the section has. Valid values: '1' or '2'.
      columns: '1'
      view: showcase
      # For Showcase view, flip alternate rows?
      flip_alt_rows: false
  #- block: markdown
  #  content:
  #    title: Gallery
  #    subtitle: ''
  #    text: |-
  #      {{< gallery album="demo" >}}
  #  design:
  #    columns: '1'
  #- block: collection
  #  id: featured
  #  content:
  #    title: Featured Publications
  #    filters:
  #      folders:
  #        - publication
  #      featured_only: true
  #  design:
  #    columns: '2'
  #    view: card
  - block: experience
    content:
      title: Recent Manuscripts
      text: |2-
          <u>Bratchikov S.</u>, L. Luo, S. Swaminathan, Y. Sokolenko, C Gao, E. Diaz, E. Jonasson, L. Luo, Z. Yu, A. Agrawal, RG Wudnerink, AV Misharin (manuscript)
          *Machine learning classifier for automated and scalable analysis of clinical flow cytometry samples*
    design:
      columns: '2'
      view: citation
  - block: experience
    id: posters
    content:
      title: Posters & Talks 
      text: |-
        ## Posters
  
        ### [Machine Learning Classifier for Automated and Scalable Analysis of Clinical Flow Cytometry Samples](https://stanislavbratchikov.github.io/uploads/Bratchikov_poster.pdf)
        - Presented at: 2023 Systems Biology for Infectious Diseases Annual Conference, Northwestern University
        - Presented at: 2023 Molecular Mechanisms of Lung Disease Annual Meeting, University of Helmholtz - Northwestern University Conference
        - Presented at: 2023 The 17th Annual Lewis Landsberg Research Day, Northwestern University
  
        ## Talks
  
        ### OpenBio 2022
        - Topic: RNA Expression-Level Heterogeneity in Cancer-Associated Stromal Cell Populations within the Tumor Microenvironment.

        
    design:
      columns: '2'
      view: citation

  #- block: tag_cloud
  #  content:
  #    title: Popular Topics
  #  design:
  #    columns: '2'
  - block: contact
    id: contact
    content:
      title: Contact
      subtitle:
      text: |-
        Feel free to contact via email:
      # Contact (add or remove contact options as necessary)
      email: stanislav.bratchikov@northwestern.edu
      address:
        street: 303 E. Superior St.
        city: Chicago
        region: IL 
        postcode: '60611'
        country: United States
      contact_links:
        - icon: twitter
          icon_pack: fab
          name: DM Me
          link: 'https://twitter.com/brtchkvs'
      # Automatically link email and phone or display as text?
      autolink: true
      # Email form provider
      form:
        provider: netlify
        formspree:
          id:
        netlify:
          # Enable CAPTCHA challenge to reduce spam?
          captcha: false
    design:
      columns: '2'
---
